### DISCUSSION TITLE
Sidebar item 'static' was not found.

### DISCUSSION THREAD

**Discussion by Ravencentric** [retypeapp/retype#660] [Q&A] [answered]
```
> retype start . --port 5555
INPUT: C:\Users\raven\Documents\GitHub\thewiki\docs
OUTPUT: [in-memory]
WARNING: Sidebar item 'static' was not found.
```

I get the above vague warning and don't know how to fix it. `--verbose` doesn't say anything more either.
Here's the repo if you want to reproduce <https://github.com/Snaacky/thewiki>

**Comment by geoffreymcgill** [comment, marked as answer]
The `/docs/static/index.yml` is not required and can be removed. By default, the `static` folder is not created as a menu item. 

We will investigate why that warning happens when the `index.yml` is present. Should be an easy fix to make. For now as a work around, just remove the `index.yml` file. 

Hope this helps.

**Reply by Ravencentric** [reply]
Thank you!
